### PR TITLE
Docs: Fine-grained access control refactor

### PR DESCRIPTION
**Pull request by chri2547**
Refactors FGAC docs.

### CODE REVIEW

File: docs/sources/enterprise/access-control-refactor/about-fgac.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- [Roles]({{< relref "../../administration/manage-users-permissions/_index.md" >}})
+- [Settings]({{< relref "../settings-updates.md" >}})
+- [Service accounts]({{< relref "../../administration/service-accounts/_index.md" >}})
+- Statistics
+- [Teams]({{< relref "../../administration/manage-users-permissions/manage-teams/_index.md" >}})
+- [Users]({{< relref "../../administration/manage-users-and-permissions/manage-server-users/_index.md" >}})
+
+To learn more about the permissions associated with each fixed role, refer to [Fine-grained access control fixed role definitions]({{< relref "./rbac-fixed-role-definitions.md" >}}).
+
+To learn how to assign fixed roles to a user or team, refer to [Assign and manage fixed roles]({{< relref "./assign-manage-fixed-roles.md" >}})
```
Comment (brendamuir) [inline docs/sources/enterprise/access-control-refactor/about-fgac.md:76]: One quick question here--is the standard to use "to learn more about", "to learn how to" or "for more information on" - just a question for my learning, really. It's an easy sentence to standardize across docs, too.

File: docs/sources/enterprise/access-control-refactor/provision-custom-roles/assigning-custom-roles.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      - action: 'users:create'
+        scope: 'users:*'
+    builtInRoles:
+      - name: 'Editor'
+      - name: 'Admin'
+```
+
+## Assign a custom role to a team
+
+Need information here on why a user would want to complete this task. What are the benefits?
```
Comment (brendamuir) [inline docs/sources/enterprise/access-control-refactor/provision-custom-roles/assigning-custom-roles.md:70]: same as above.I'm guessing you put these questions in for the review? that's good to know in that case that we can work like that!

**Review by brendamuir — CHANGES_REQUESTED** [review]
Hey Chris,I'm not sure I haven't just entered annoying comments here-but it was great to read through for learning-and would love to chat through it in our call later!!

File: docs/sources/enterprise/access-control-refactor/rbac-fixed-role-definitions.md [hunk 3]
```diff
@@ -54,14 +56,3 @@ The following table lists permissions associated with each fixed role.
...
 | `fixed:annotations:writer`             | `annotations:write` <br>`annotations.create`<br> `annotations:delete` for scope `annotations:type:*`                                                                                                                                                                     | Create, update and delete all annotations and annotation tags.                                                                                                                                                                                                                        |
-
-## Built-in role assignments
```
Comment (gguillotte-grafana) [inline docs/sources/enterprise/access-control-refactor/rbac-fixed-role-definitions.md]: Did this move somewhere else, or is it being removed?

Comment (chri2547) [reply docs/sources/enterprise/access-control-refactor/rbac-fixed-role-definitions.md]: I moved that content and neglected to remove the heading. All done now, thanks!

File: docs/sources/enterprise/settings-updates.md [hunk 4]
```diff
@@ -85,7 +85,7 @@ settings updates. If there are updates, it reloads the Grafana services affected
 The background job synchronizes settings between instances in high availability set-ups. So, after you perform some changes through the
 HTTP API, then the other instances are synchronized through the database and the background job.
 
-## Control access with fine-grained access control
+## Control access with role-based access control
 
-If you have [Fine-grained access Control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, you can control who can read or update settings.
+If you have [Role-basedaccess control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, you can control who can read or update settings.
```
Comment (mjseaman) [inline docs/sources/enterprise/settings-updates.md:90]:
```suggestion
If you have [Role-based access control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, you can control who can read or update settings.
```

**Review by mjseaman — CHANGES_REQUESTED** [review]
Just a couple of edits and a couple of notes, looking gooood

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
-A permission is an action and a scope. When creating a fine-grained access control, consider what specific action a user should be allowed to perform, and on what resources (its scope).
+A permission is comprised of an action and a scope. When creating a custom role, consider the actions the user can perform and the resource on which to perform them.
 
-To grant permissions to a user, you create a built-in role assignment to map a role to a built-in role. A built-in role assignment _modifies_ to one of the existing built-in roles in Grafana (Viewer, Editor, Admin). For more information, refer to [Built-in role assignments]({{< relref "./roles.md#built-in-role-assignments" >}}).
-
-To learn more about which permissions are used for which resources, refer to [Resources with fine-grained permissions]({{< relref "./_index.md#resources-with-fine-grained-permissions" >}}).
+To learn more about the Grafana resources to which you can apply RBAC, refer to [Resources with RBAC permissions]({{< relref "./about-rbac.md#fixed-roles" >}}).
 
 action
-: The specific action on a resource defines what a user is allowed to perform if they have permission with the relevant action assigned to it.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/enterprise/access-control/custom-role-actions-scopes.md:17]: Agreed, fixed this.

**Review by brendamuir — APPROVED** [review]

**Review by mjseaman — APPROVED** [review]
LGTM!

File: docs/sources/enterprise/access-control/enable-rbac-and-provisioning.md [hunk 6]
```diff
@@ -63,6 +63,9 @@ This section describes how to enable RBAC by setting a feature flag or adding an
 
 ## Enable role provisioning
 
+You can create, change or remove [Custom roles]({{< relref "./roles.md#custom-roles" >}}) and create or remove [built-in role assignments]({{< relref "./roles.md#built-in-role-assignments" >}}), by adding one or more YAML configuration files in the [`provisioning/access-control/`]({{< relref "../../administration/configuration/#provisioning" >}}) directory.
```
Comment (chri2547) [inline docs/sources/enterprise/access-control/enable-rbac-and-provisioning.md:66]: @mjseaman, how did this content get in here? The relrefs are broken and the "built-in" is now called "basic".

**Pull request merged**